README.md

```diff
@@ -153,8 +153,8 @@ server {
 }
 ```
 
-Prior to using MySQL, MongoDB or SQLite you need to create the database and tables using [one of the provided files](db).  
```

Comment:
Ah. I just kinda mindlessly so `:%s/ . *$//` when I notice trailing whitespace in a file due to experience with some other, let's say less careful, developers. I'll put those back, sorry.

Was unaware that was a thing in markdown. Honestly if it was me I'd just put an explicit `<br />` in there or a `\n\n` just because I can't see trailing spaces. I'll see about changing my vim config to make those visible if that's a thing people do.